### ISSUE TITLE
datetime.datetime.fromtimestamp(0) fails in python 3.6 on windows

### ISSUE THREAD

**Issue by motir** [home-assistant/appdaemon#83]
dashboard.py line 456:
late_file = datetime.datetime.fromtimestamp(0)

throws:
```
2017-07-13 22:45:45.973677 WARNING Traceback (most recent call last):
  File "c:\python3\lib\site-packages\appdaemon\appdash.py", line 93, in _load_dash
    dash = dashboard.compile_dash(name, skin, skindir, request.rel_url.query)
  File "c:\python3\lib\site-packages\appdaemon\dashboard.py", line 26, in profiled_fn
    dash = fn(*args, **kwargs)
  File "c:\python3\lib\site-packages\appdaemon\dashboard.py", line 44, in newfunc
    result = func(*args, **kwargs)
  File "c:\python3\lib\site-packages\appdaemon\dashboard.py", line 495, in compile_dash
    widget_mod = latest_file(os.path.join(conf.dash_dir, "widgets"))
  File "c:\python3\lib\site-packages\appdaemon\dashboard.py", line 456, in latest_file
    late_file = datetime.datetime.fromtimestamp(0)
OSError: [Errno 22] Invalid argument

```

This is documented as a bug in Python 3.6 (tested with 3.6.2 rc2 and it still exists).

Recommended solution:
Use utcfromtimestamp instead of fromtimestamp:

late_file = datetime.datetime.utcfromtimestamp(0)

**Comment by acockburn** [comment]
Thanks - I'll take a look at fixing this in the next release.

**Comment by acockburn** [comment]
Looking more deeply, this isn;t really a universal fix. I use this call all over appdaemon and sometimes I am relying on the semantics of fromtimestamp vs utcfromtimestamp - I can't simply replace all instances. I'm afraid we will have to wait for a fix in 3.6.

**Comment by motir** [comment]
Ok, then another idea:

I did some research and found that this exception only occurs when you provide a number that is less than 86400 (which is 1 day after 1/1/1970). It will not happen if you provide bigger numbers.

Then just change the 0's in your code (lines #456 and #490) to 86400 and it should be resolved.
